packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -131,6 +86,9 @@ export const CallControls = (props: CallControlsProps): JSX.Element => {
   );
 };
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const isEnabled = (option: any): boolean => option !== false;
```

Comment:
option can only be boolean here
any particular reason for using `any`?

Reply:
typescript recommends to use `unknown` instead of `any` for these scenarios

Reply:
Oh right `unknown` still allows me to do an eqeqeq check. Done.